commands/team_e2e_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Error renaming team which does not exist", func(c client.Client) {
+		printer.Clean()
+		oldTeamName := "existingName"
+		cmd := &cobra.Command{}
+		args := []string{""}
+		args[0] = oldTeamName
```

Comment:
I'd probably name this `nonExistentTeamName` to make it more clear.
Also we can compress a bit the initialization:

```
args := []string{"nonExistentTeamName"}
```